### PR TITLE
20231218-1551

### PR DESCRIPTION
**Pull request by UTCGilligan**
For first production deployment. Already deployed to https://gotest.utc.edu.

### CODE REVIEW

File: frontend/assets/svg/utc-wordmark-reverse.svg
Comment (UTCGilligan) [inline frontend/assets/svg/utc-wordmark-reverse.svg:1]: This can't be viewed because one of the lines in the SVG code is too long for Github to parse.

**Review by UTCGilligan — COMMENTED** [review]
Looks good, it's been deployed to https://gotest.utc.edu for review.

**Review by UTCGilligan — COMMENTED** [review]
OK, looks good again. Just a few tweaks to things that nobody has noticed thus far:
- 1px tweak to height of img.logo on backend
- Close button styling for the alert when a reserved/used shortcut  is tried again

**Review by UTCbridget — APPROVED** [review]
Fantastic work, Chris! Look forward to seeing in production.

**Pull request merged**